docs/pages/includes/permission-warning.mdx

```diff
@@ -1,5 +1,6 @@
-<Details title="Best practices for production security">
-
+<Admonition type="important">
+##### Best practices for production security
```

Comment:
Here is what this looks like the [CLI reference index page](https://docs-arahiy3cq-goteleport.vercel.app/docs/reference/cli/cli/):

<img width="741" alt="image" src="https://github.com/user-attachments/assets/34751094-341c-4576-b434-edb8b171a31a">